app/views/admin/cfps/_booths_cfp.html.haml

```diff
@@ -7,6 +7,10 @@
 %dd
   = @cfp.end_date.strftime('%A, %B %e. %Y')
 %dt
+  Description
+%dd
+  = markdown(truncate(@cfp.description))
```

Comment:
If we truncate this here in the show action, where are we showing the whole text on the admin side?

Reply:
Yep I should not have added truncate for show actions.😅😅